src/utill.rs

```diff
@@ -287,9 +287,9 @@ pub(crate) fn check_and_apply_maker_private_keys<S: SwapCoin>(
 ///
 /// the Maker's advertised Pubkey with these two nonces.
 #[allow(clippy::type_complexity)]
+/// Generate maker keys from the Maker's advertised Pubkey with nonces.
 pub(crate) fn generate_maker_keys(
     tweakable_point: &PublicKey,
-    count: u32,
```

Comment:
I am having a second thought on this. We may need this count later when doing utxo splitting. So for that future sake we can keep this for now. 

keep the function as it is like before.